diff --git a/src/main/java/net/minecraft/client/gui/inventory/InventoryScreen.java b/src/main/java/net/minecraft/client/gui/inventory/InventoryScreen.java
index 7a2c11e..7c3138e 100644
--- a/src/main/java/net/minecraft/client/gui/inventory/InventoryScreen.java
+++ b/src/main/java/net/minecraft/client/gui/inventory/InventoryScreen.java
@@ -23,3 +23,3 @@ public class InventoryScreen extends AbstractContainerScreen {
 
-		final int var3;
+		int var3;
 		int var4;
@@ -32,5 +32,6 @@ public class InventoryScreen extends AbstractContainerScreen {
 		for(var3 = 0; var3 < 4; ++var3) {
-			this.slots.add(new ContainerSlot(this, var1, var1.getContainerSize() - 1 - var3, 8, 8 + var3 * 18) {
+			int finalVar = var3;
+			this.slots.add(new ContainerSlot(this, var1, var1.getContainerSize() - 1 - finalVar, 8, 8 + finalVar * 18) {
 				public boolean mayPlace(ItemInstance var1) {
-					return var1.getItem() instanceof ArmorItem ? ((ArmorItem)var1.getItem()).slot == var3 : false;
+					return var1.getItem() instanceof ArmorItem ? ((ArmorItem)var1.getItem()).slot == finalVar : false;
 				}
@@ -38,3 +39,3 @@ public class InventoryScreen extends AbstractContainerScreen {
 				public int getNoItemIcon() {
-					return 15 + var3 * 16;
+					return 15 + finalVar * 16;
 				}
